cypress/integration/journalistCanCreateArticles.feature.js

```diff
@@ -1,96 +1,98 @@
 /* eslint-disable no-undef */
-describe("Journalist can create articles", () => {
+describe('Journalist can create articles', () => {
   beforeEach(() => {
     cy.server();
     cy.route({
-      method: "POST",
-      url: "http://localhost:3000/api/auth/sign_in",
-      response: "fixture:journalist_can_login.json",
+      method: 'POST',
+      url: 'http://localhost:3000/api/auth/sign_in',
+      response: 'fixture:journalist_can_login.json',
       headers: {
-        uid: "journalist@mail.com",
+        uid: 'journalist@mail.com',
       },
     });
     cy.route({
-      method: "GET",
-      url: "http://localhost:3000/api/auth/validate_token**",
-      response: "fixture:journalist_can_login.json",
+      method: 'GET',
+      url: 'http://localhost:3000/api/auth/validate_token**',
+      response: 'fixture:journalist_can_login.json',
     });
     cy.route({
-      method: "POST",
-      url: "http://localhost:3000/api/articles",
-      response: { message: "Your article was successfully created!" },
+      method: 'POST',
+      url: 'http://localhost:3000/api/articles',
+      response: { message: 'Your article was successfully created!' },
     });
-    cy.visit("/");
+    cy.visit('/');
   });
 
-  it("successfully", () => {
+  it('successfully', () => {
+    const title = 'Covid vaccine is found';
```

Comment:
the test is changed to use a const for the title, why is that? Should it be changed for subtitle and body also for consistency?

Reply:
Hmm good catch, will either def all typed inputs as constants or none of them

Reply:
oh!! We will remove that const.. it was me who played around to test the image thingy. Thanks.